logistics_planning_invoicing/models/logistics_schedule.py

```diff
@@ -147,6 +147,25 @@ def action_create_invoice_wizard(self):
             'type': 'ir.actions.act_window',
         }
 
+    def action_add_existing_invoice_wizard(self):
+        ls_ids = self.browse(self._context['active_ids'])
+        if not all(ls.carrier_id for ls in ls_ids) or len(ls_ids.mapped('carrier_id')) > 1:
+            raise ValidationError(_('There are different carriers on the selected schedules or there is no carrier.'))
```

Comment:
We should add more restrictions (the same for "Add to existing invoice" form button invisible domain):
* Action shouldn't be accessible by non-account users => is it possible to directly add this restriction to action definition, rather than a python check?
* Schedule must be marked as invoiceable.
* Schedule shouldn't be marked as done neither linked to an invoice line.

And also those restrictions added for "Create new invoice" batch process (stock move filled, has quantity, is finished):
https://github.com/solvosci/slv-stock/blob/5cf2bb953793884af7a91fbe8c3f946fe740a3fb/logistics_planning_invoicing/models/logistics_schedule.py#L121-L129